main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gorilla/mux"
+)
+
+// Movie struct
+type Movie struct {
+	ID       int    `json:"ID"`
+	Title    string `json:"Title"`
+	Director string `json:"Director"`
+}
+
+// Movies list
+var Movies []Movie
+
+func getMovies(w http.ResponseWriter, r *http.Request) {
+	csvFile, err := os.Open("movies.csv")
```

Comment:
This is not a task of the controller layer. This looks like a task the service layer should take care of.